core/domain/expression_parser_test.py

```diff
@@ -70,11 +70,11 @@ def test_is_algebraic(self):
         with self.assertRaises(Exception):
             expression_parser.is_algebraic('1 +2)')
         with self.assertRaises(Exception):
-            expression_parser.is_algebraic('Alpha')
+            expression_parser.is_algebraic('a~2')
```

Comment:
I've only removed tests which are obsolete now. Now, the expression `Alpha` will not raise an error but will be interpreted as  `A*l*p*h*a`.

Reply:
Then still move it to its relevant place (e.g. a test that tokenizes Alpha to A, l, p, h, a) to show what happens in that scenario. You can still also have this be in is_algebraic() too, it would return True presumably.

Ditto for other tests, unless there's already a carbon-copy in the existing tests. But presumably those tests were originally added because they provided inputs that were different from the existing tests, and the change in the implementation and resulting output doesn't change that.

Reply:
Done.